policyengine_us/parameters/calibration/SSI/payments.yaml

```diff
@@ -0,0 +1,111 @@
+description: Total amount of federally administered SSI payments for 2021 by state or other area
+metadata: 
+  unit: currency-USD
+  period: Calendar Year 2021
+  reference:
+  - title: Supplemental Security Income State Data | Social Security Administration
+    href: https://www.ssa.gov/policy/docs/statcomps/supplement/2022/7b.html#table7.b1
+U.S. Totals:
```

Comment:
Could we make this nested (and with the relevant metadata- not for each state of course by just the nodes), e.g:
* Federal
* By State
  * AL
  * AK
  * ...

Reply:
Also- if you can find projections for SSI payments in the future, even at the federal level only, we can add a bit of metadata to uprate State SSI payments by the federal increases.